styleguide/STYLEGUIDE.md

```diff
@@ -1,26 +1,31 @@
-## General
+## DuckDuckGo Android Style Guide
 * We care about clean code and aim to make this codebase as self-documenting and readable as possible.
-* We primarily use Kotlin and follow the standard coding conventions outlined in https://kotlinlang.org/docs/reference/coding-conventions.html
+* We primarily use Kotlin and follow coding conventions based on the [Android Kotlin Style Guide](https://developer.android.com/kotlin/style-guide)
 * There may be instances of code that pre-dates our use of this style guide, these can be refactored as we encounter them.
 
-**IDE Setup:**
-* Open Android Studio preferences
-* Tap on code style, select Kotlin
-* Select Set From, Predefined Style, Kotlin Style Guide
-![Setting kotlin style screenshot](IDE_kotlin_style.png)
+###  Android Studio Setup
 
-## Line breaks
-* We use a line margin of 150 rather than the default of 100 as the default causes excessive line breaks. The larger margin allows us to take advantage of modern widescreen monitors with more screen real-estate.
+It is possible to get Android Studio to adhere to these style guidelines, via a rather complex sequence of menus. To make it easier, we've provided a coding style that can be imported into Android Studio.
+The file can be found [here](ddg-settings.zip).
 
-**IDE Setup:**
-* Open android studio preferences
-* Tap on code style
-* Enter 150 into "Page margin"
+* Open Android Studio File -> Settings
 
-![Setting kotlin style screenshot](IDE_line_wrap.png)
+![Setting kotlin style screenshot](import_settings_one.png)
 
+* Select the file you just downloaded
 
-## Timber
+
+![Setting kotlin style screenshot](import_settings_two.png)
+
+* Import all components required (AndroidEditors and Code Style)
+
+![Setting kotlin style screenshot](import_settings_three.png)
```

Comment:
I'd also consider removing this screenshot; already the UI differs between your screenshot and what I see. I think your text instructions are sufficient.